sm_project/cl_app/admin.py

```diff
@@ -3,8 +3,8 @@
 from .models import Checklist, ChecklistItem
 
 
-# Creates a list filter that filters by specific users and displays which both the user's name and email 
 class UserFilter(admin.SimpleListFilter):
+    # Creates a list filter that filters by specific users and displays which both the user's name and email 
```

Comment:
ok sorry to be overly pedantic BUT!
This is a comment, not a docstring.

a docstring is like this

```python
def my_func():
    """
    docstring
    """
   pass
```

this is actually important, because within the console if you use `help(my_func) it will print the docstring, but not the comment. Also, automatic documentation generators will read the docstring, but not the comment.